README.md

```diff
@@ -245,6 +249,7 @@ The regression tests can be run automatically by running the script `scripts/Run
 ```
 $ ./scripts/RunRegressionTests 4 9
 ```
+The default option uses local data to perform the run. The addition of a "-d" flag will switch the regression test to use the data pipeline locally stored test data instead.
```

Comment:
I just realised that this also requires setting of the PYTHONPATH to include the data_pipeline_api directory. Maybe worth adding a mention of this here (or elsewhere), as otherwise it results in some cryptic error messages.